src/coverlet.msbuild.tasks/coverlet.msbuild.tasks.csproj

```diff
@@ -1,18 +1,63 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net472</TargetFrameworks>
     <AssemblyVersion>2.4.0</AssemblyVersion>
+
+    <PackageId>coverlet.msbuild</PackageId>
+    <PackageVersion>2.5.0</PackageVersion>
+    <Title>coverlet.msbuild</Title>
+    <Authors>tonerdo</Authors>
+    <PackageLicenseUrl>https://github.com/tonerdo/coverlet/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>http://github.com/tonerdo/coverlet</PackageProjectUrl>
+    <PackageIconUrl>https://nuget.org/Content/gallery/img/default-package-icon.svg</PackageIconUrl>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <DevelopmentDependency>true</DevelopmentDependency>
+    <Description>Coverlet is a cross platform code coverage library for .NET Core, with support for line, branch and method coverage.</Description>
+    <PackageTags>coverage testing unit-test lcov opencover quality</PackageTags>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <ContentTargetFolders>build</ContentTargetFolders>
+
+    <!-- Build tasks should not be added to the lib folder. -->
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <IncludeSymbols>true</IncludeSymbols>
+    <IncludeSources>true</IncludeSources>
+
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);PackBuildOutputs</TargetsForTfmSpecificContentInPackage>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="ConsoleTables" Version="2.1.0" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.5.180" />
+    <PackageReference Include="ConsoleTables" Version="2.1.0" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.5.180" PrivateAssets="All" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\coverlet.core\coverlet.core.csproj" PrivateAssets="All" />
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="$(MSBuildThisFileDirectory)..\coverlet.core\coverlet.core.csproj" />
+    <None Include="..\coverlet.msbuild\coverlet.msbuild.props">
+      <Link>build\coverlet.msbuild.props</Link>
+      <Pack>true</Pack>
+    </None>
+    <Content Include="..\coverlet.msbuild\coverlet.msbuild.targets">
```

Comment:
is it intentional that you are using a Content Element here but for the props a None element? Also for the props you set `Pack` to true but not here?

Reply:
Not intentional. I was experimenting and found that both worked, but forgot to settle on one. 😆

Reply:
Resolved this along with https://github.com/tonerdo/coverlet/pull/299#issuecomment-454998349 in 1f6c7ea6b36cf07e68eeb643ec4d35a9a274cf7f.